include/re/re.h

```diff
@@ -155,7 +149,7 @@ re_perror(enum re_dialect dialect, const struct re_err *err,
  */
 int re_sgetc(void *opaque); /* expects opaque to be char ** */
 int re_fgetc(void *oapque); /* expects opaque to be FILE *  */
-int re_bgetc(void *opaque); /* expects opaque to be struct re_buf * */
+int re_bgetc(void *opaque); /* expects opaque to be struct string * from adt/string.h */
```

Comment:
But it is? For `re_fgetc`, you have to `#include <stdio.h>`, for `re_bgetc` you have to `#include <adt/string.h>`.